frontend/src/components/ContentDisplayToggle.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import clsx from "clsx";
+
+import { useState } from "react";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+// this could implemented slightly differently to either be simpler and more generic
+// or we could opt to build the entire show / hide functionality into this component and pass the content controlled by the toggle as children
+export default function ContentDisplayToggle({
+  toggleOnText,
+  toggleOffText,
+  visibleClassname,
+  children,
+}: {
+  toggleOnText: string;
+  toggleOffText: string;
+  visibleClassname: string;
+  children: React.ReactNode;
+}) {
+  const [toggledContentVisible, setToggledContentVisible] =
+    useState<boolean>(false);
+
+  const iconName = toggledContentVisible ? "arrow_drop_up" : "arrow_drop_down";
+  return (
+    <>
+      <div className="display-flex flex-column flex-align-center tablet:display-none">
```

Comment:
Good idea! You'd always need the same breakpoint to show the content and hide the toggle. Instead of `visibleClassname="tablet:display-block"`, you could pass `breakpoint="tablet"` and concat the display classes — e.g. `${breakpoint}:display-none` and `${breakpoint}:display-block`.

Reply:
yep, I definitely missed that. I'll rework